action.yml

```
@@ -21,6 +21,8 @@ outputs:
     description: "The installed python version. Useful when given a version range as input."
   cache-hit:
     description: 'A boolean value to indicate a cache entry was found'
+  interpreter-path:
```

```suggestion
  python-path:
```

Naming it this way parallels the `python-version` output.

---

done.